modules/accredited_representative_portal/db/migrate/20241227212839_remove_old_indices_from_poa_forms.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveOldIndicesFromPoaForms < ActiveRecord::Migration[7.1]
```

Comment:
I want to note here that I brought this up in the CoP this morning, and that while it's not always necessary, we generally recommend following the same patterns for removing indexes as we do for adding (regarding transactions/concurrency). Considering there aren't any records in the table, we'll let it go here.